app/ui/node-settings/browser.js

```diff
@@ -0,0 +1,310 @@
+        $last_tag.val('').focus();
+      } else {
+        var $parent = $last_tag.parent();
+        $last_tag.remove();
+        $parent.html('<div class="tag-name">' + value + '</div><div class="fs-delete">x</div>');
+      }
+
+      var tags = $fs_prop_tags.find('li').map(function () {
+        return $(this).find('.tag-name').text();
+      }).get()
+
+      $(document).unbind('click');
+      e.preventDefault();
+    }
+
+    var allowed = new RegExp('^[a-zA-Z0-9\-\_]+$');
+    if (allowed.test(strcode) || code == 8) {
+      return true;
+    }
+
+    e.preventDefault();
+    return false;
+  }
+
+  function nodeSettingsInitMetaTagCompletion() {
+    var $meta_tags        = $('#node-settings-properties-meta-tags'),
+        $meta_tags_parent = $meta_tags.parent(),
+        $input            = $meta_tags_parent.find('input');
+
+    $input.kendoAutoComplete({
+      minLength    : 0,
+      dataTextField: '_id',
+      dataSource   : new kendo.data.DataSource({
+        transport: {
+          read: function (operation) {
+            sync.blnApi.getNodeAttributeSummary({attributes: ['meta.tags']}, (err, data) => {
+              if (err) {
+                nodeSettingsShowErrorMessage(err.message);
+              } else {
+                data['meta.tags'].sort();
+                operation.success(data['meta.tags']);
+              }
+            })
+
+          }
+        }
+      }),
+      sort         : {
+        dir  : 'asc',
+        field: '_id'
+      },
+      change       : function (e) {
+        this.dataSource.read();
+      },
+    })
+
+    $input.unbind('focus').bind('focus', function () {
+      $meta_tags.addClass('fs-select-tags');
+      $input.data('kendoAutoComplete').search();
+    })
+
+    $input.unbind('blur').bind('blur', function () {
+      $meta_tags.removeClass('fs-select-tags');
+    })
+  }
+
+  function closeNodeSettingsWindow () {
+    ipcRenderer.send('node-settings-close');
+  }
+
+  function nodeSettingsCompileTemplates (node) {
+    var templateContentHtml = $('#node-settings-template-content').html();
+    var $placeholderContent = $('#node-settings-contentWrapper');
+    var templateContent     = handlebars.compile(templateContentHtml);
+
+    var context = {}
+    if (node) {
+      var locale     = i18n.getCurrentLocale(),
+          nodeChange = new Date(node.changed.sec * 1000)
+      context        = {
+        nodePath           : node.path,
+        nodeChangeDayLong  : nodeChange.toLocaleString(locale, {weekday: 'long'}),
+        nodeChangeMonthDay : nodeChange.getDate(),
+        nodeChangeMonthLong: nodeChange.toLocaleDateString(locale, {month: 'long'}),
+        nodeChangeYear     : nodeChange.toLocaleDateString(locale, {year: 'numeric'}),
+        nodeChangeTime     : nodeChange.getHours() + ':' + (nodeChange.getMinutes() < 10? '0' : '') + nodeChange.getMinutes(),
+      }
```

Comment:
There are different locales with different date/time format. This looks pretty static.
nodeChange.toLocaleString(locale) would probably be enough and u can remove everything else.